packages/devtools/devtools/src/panels/mesh/SignalMessagesContainer.tsx

```diff
@@ -0,0 +1,30 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import React, { useEffect, useState } from 'react';
+
+import { SignalResponse } from '@dxos/protocols/proto/dxos/devtools/host';
+import { useDevtools } from '@dxos/react-client';
+
+import { SignalMessages } from './SignalMessages';
+
+const SignalMessagesContainer = () => {
```

Comment:
@mykola-vrmchk this is `SignalMessagesContainer` the data-loading wrapper for the pure `SignalMessages` (note pure doesn't mean it can't encapsulate private state, just means it's separate from the "app-specific" data environment)